app/controllers/api/v1/organisations_users_controller.rb

```diff
@@ -29,7 +29,7 @@ class OrganisationsUsersController < Api::V1::ApiController
       api :POST, "/v1/organisations_user", "Create a package"
       param_group :organisations_user
       def create
-        builder = OrganisationsUserBuilder.new(params['organisations_user'].to_hash).build
+        builder = OrganisationsUserBuilder.new(params['organisations_user'].to_hash).build(is_stock_app?)
```

Comment:
Rather than passing `is_stock_app?` as an argument to the `build` method, I think it would be better to pass the `app_name` to the `initializer` to make it more generic and adaptable to future needs. For example, we might need app_name in the future for a different reason.

e.g. `OrganisationsUserBuilder.new(params: params['organisations_user'].to_hash, app_name: app_name).build`